From 7998fd0238460fd4b8857e5a0c4cc066db4a979e Mon Sep 17 00:00:00 2001
From: Elvis Pranskevichus <elvis@magic.io>
Date: Sat, 24 Mar 2018 20:31:06 -0400
Subject: [PATCH 08/10] system nodejs

---
 third_party/node/node.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/third_party/node/node.py b/third_party/node/node.py
index 0194f56155..52e210140a 100755
--- a/third_party/node/node.py
+++ b/third_party/node/node.py
@@ -10,11 +10,7 @@ import sys
 
 
 def GetBinaryPath():
-  return os_path.join(os_path.dirname(__file__), *{
-    'Darwin': ('mac', 'node-darwin-x64', 'bin', 'node'),
-    'Linux': ('linux', 'node-linux-x64', 'bin', 'node'),
-    'Windows': ('win', 'node.exe'),
-  }[platform.system()])
+  return '/usr/bin/node'
 
 
 def RunNode(cmd_parts, stdout=None):
-- 
2.19.0

