From b1d41179c11965a8826c78aa95563233c00e9213 Mon Sep 17 00:00:00 2001
From: Elvis Pranskevichus <elvis@magic.io>
Date: Sun, 29 Oct 2017 16:38:40 -0400
Subject: [PATCH 1/2] system-icu support

---
 electron.gyp | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/electron.gyp b/electron.gyp
index 8e5067b93..261d89428 100644
--- a/electron.gyp
+++ b/electron.gyp
@@ -242,7 +242,6 @@
               'files': [
                 '<@(copied_libraries)',
                 '<(libchromiumcontent_dir)/locales',
-                '<(libchromiumcontent_dir)/icudtl.dat',
                 '<(libchromiumcontent_dir)/blink_image_resources_200_percent.pak',
                 '<(libchromiumcontent_dir)/content_resources_200_percent.pak',
                 '<(libchromiumcontent_dir)/content_shell.pak',
@@ -302,6 +301,9 @@
         '.',
         'chromium_src',
         'native_mate',
+        # libicu headers shim.
+        '<(SHARED_INTERMEDIATE_DIR)/shim_headers/icui18n_shim',
+        '<(SHARED_INTERMEDIATE_DIR)/shim_headers/icuuc_shim',
         # Include atom_natives.h.
         '<(SHARED_INTERMEDIATE_DIR)',
         # Include directories for uv and node.
-- 
2.21.0

